docker-compose.yml

```diff
@@ -27,12 +27,12 @@ services:
   api:
     env_file:
       - .env
-    image: postgrest/postgrest
+    image: postgrest/postgrest:v0.4.4.0
     ports:
       - 3000:3000
 
   db:
-    image: postgres:10.3-slim
+    image: postgres:latest
```

Comment:
I'd rather remove this change from this particular PR because:

1. This has nothing to do with the PR title
2. The other change in this very same file was pinning a Docker Image version; and this is basically unpinning; we need to document why pinning that and unpinning this – and this is why a PR dedicated to this particular issue can be helpful

Reply:
(Might be related to my recent commit 464b62a0cd1baaf0accac12abd63885cafbf0af2)